build/gh-pages-app.js

```diff
@@ -7,6 +7,12 @@
 
 const fs = require('fs');
 const path = require('path');
+const rollup = require('rollup');
+const commonjs =
+  // @ts-expect-error types are wrong.
+  /** @type {import('rollup-plugin-commonjs').default} */ (require('rollup-plugin-commonjs'));
+const {terser: rollupTerser} = require('rollup-plugin-terser');
+const {nodeResolve} = require('@rollup/plugin-node-resolve');
```

Comment:
I think you need to add this to our deps

https://github.com/GoogleChrome/lighthouse/pull/12878/checks?check_run_id=3267146690#step:6:32